### PR TITLE
Add PopeenCom.ClassicVolumeMixer 2.2

### PR DESCRIPTION
**Pull request by popeen**
- [x ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/97832)

### CODE REVIEW

**Comment by popeen** [comment]
@microsoft-github-policy-service agree [company="PopeenCom"]

**Comment by popeen** [comment]
@microsoft-github-policy-service agree

File: manifests/p/PopeenCom/ClassicVolumeMixer/2.2/PopeenCom.ClassicVolumeMixer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: PopeenCom.ClassicVolumeMixer
+PackageVersion: 2.2
+PackageLocale: en-US
+Publisher: PopeenCom
+PackageName: Classic Volume Mixer
+License: MIT License
+ShortDescription: Classic Volume Mixer Setup
```
Comment (Trenly) [inline manifests/p/PopeenCom/ClassicVolumeMixer/2.2/PopeenCom.ClassicVolumeMixer.locale.en-US.yaml:10]: Please provide a better short description of what Classic Volume Mixer does

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/p/PopeenCom/ClassicVolumeMixer/2.2/PopeenCom.ClassicVolumeMixer.locale.en-US.yaml (hunk 1 above)
Comment (popeen) [reply manifests/p/PopeenCom/ClassicVolumeMixer/2.2/PopeenCom.ClassicVolumeMixer.locale.en-US.yaml]: Done

**Review by stephengillie — APPROVED** [review]

**Pull request merged**